tests/functional/test_matrix_service.py

```diff
@@ -0,0 +1,117 @@
+import unittest
+import os
+import requests
+import json
+
+import s3fs
+import zarr
+import numpy
+
+from .wait_for import WaitFor
+from matrix.common.constants import MatrixRequestStatus
+from matrix.common.pandas_utils import convert_dss_zarr_root_to_subset_pandas_dfs
+
+
+INPUT_BUNDLE_IDS = {
```

Comment:
Should we use a test fixtures bucket to store and load test bundles from to ensure test consistency across environments?

Reply:
Not a concern now, but will take note for the future